src/cmake/thirdparty/SetupGeosxThirdParty.cmake

```diff
@@ -108,6 +108,7 @@ endif()
 ################################
 # HDF5
 ################################
+set(HDF5_DIR ${GEOSX_TPL_DIR}/hdf5)
```

Comment:
I don't think that this file should be modified for a specific build. As @TotoGaz points out, defining the variable pretty much invalidates the if statement that follows the definition...which is bad.

Reply:
Solved by including tpls.cmake in pangea-base